nvtabular/io.py

```diff
@@ -612,17 +614,21 @@ def clean_pw_cache():
         return
 
 
+def close_cached_pw(fs):
+    md_dict = {}
+    with get_cache() as cache:
+        for path, (pw, bio) in cache.pq_writer_cache.items():
+            fn = bio.split(fs.sep)[-1]
```

Comment:
Can't bio be a bytes IO object if 'mem=True' when calling get_pq_writer? Should we use path instead?

Reply:
I can add an assertion, but this function should never be called for the "full" shuffle. bio will always be a path here.